### ISSUE TITLE
Question: Honebee.Core references location?

### ISSUE THREAD

**Issue by michaldengusiak** [ladybug-tools/honeybee-revit#11]
@ksobon 

I guess too early to tests...as do not want to disturb your development ....

when I try to build a solution I have a warning regarding references,  I have installed 2020 and 2021
where do you store references from Honebee.Core? 

![image](https://user-images.githubusercontent.com/26113670/85293608-8011d300-b495-11ea-91e0-d42e0365655f.png)

I can see it was installed and Open in Revit 2020, but got some errors I guess related to build
![image](https://user-images.githubusercontent.com/26113670/85294103-24941500-b496-11ea-8b35-c43a9aec187d.png)

also missing Level00
![image](https://user-images.githubusercontent.com/26113670/85294391-789ef980-b496-11ea-84e7-348e48cee87e.png)

**Comment by ksobon** [comment]
Hi Michal,

So from the top:
- Honeybee.Core doesn't have any RevitAPI dependencies. This must be an old artifact. I will clean that up, but it should build fine. 
- Regarding an issue with potentially missing Levels. I am not 100% sure that's an issue as it would only collect Rooms/Spaces that are placed (Area > 0). If it happens that there is no Space on a given level, then that level won't even show up in a list. 
- Finally, the crash is happening because there is a step in the installation process, that's missing. Honeybee.Revit needs Ladybug Tools deployed. @chriswmackey posted instructions for how to get that installed: https://github.com/ladybug-tools/honeybee-revit/issues/6#issuecomment-646849918 I will update the Readme to reflect these instructions. That should stop the crashes from happening when you run the simulation. Eventually this will all be rolled into an installer.

**Comment by michaldengusiak** [comment]
ok great thanks,

1. ok, thanks ... will ignore now and test when the update will arrive
2. this is correct in the model 
![image](https://user-images.githubusercontent.com/26113670/85299342-0e3d8780-b49d-11ea-8638-2982f8723633.png)
3. I just installed the latest dragonfly
![image](https://user-images.githubusercontent.com/26113670/85299393-1eedfd80-b49d-11ea-8946-15518bf6819b.png)

Please get my latest t[est model template here](https://github.com/HoareLea/SAM_samples/tree/master/Revit)


ps. still is crushing for me
![image](https://user-images.githubusercontent.com/26113670/85299543-4cd34200-b49d-11ea-8715-21d8e1f0059a.png)

**Comment by ksobon** [comment]
@michaldengusiak thanks for the model. I will test and get back to you.

**Comment by ksobon** [comment]
@michaldengusiak so this is one tricky model! I haven't tried parsing a bunch of things that this model has. It might be a while before we can process this thing with a level of consistency. For now, I got this: 

![image](https://user-images.githubusercontent.com/529781/85302956-76757480-b46f-11ea-8fba-b73d3185104b.png)

So we are able to get most of the rooms except: 
- Atrium 01025
- Attic 00901

These two are throwing exceptions so I will have to work on that. 

Also, bunch of these different curtain wall systems are not coming through. We are getting windows pretty well though. Of course that Roof is SOMETHING. I am not sure if we will ever be able to get that. :-) I think I might have to focus on the easier to tackle things first. Things like Curtain Walls, and leave that Roof for later. @mostaphaRoudsari @chriswmackey just for reference so you know what I am talking about: 

![image](https://user-images.githubusercontent.com/529781/85303478-277c0f00-b470-11ea-96b0-2a00ffa171d2.png)

Anyways, thanks for sharing. I will test it out some more. 

Ps. @chriswmackey can you help @michaldengusiak with the installation process? It sounds like he's having issues.

**Comment by mostaphaRoudsari** [comment]
@michaldengusiak, can you open a new issue with installation and provide more details on the installation issues?

**Comment by ksobon** [comment]
@michaldengusiak before you open a new issue. I can see that you downloaded the Dragonfly Installer. The steps ask for Ladybug Tools to be installed. That's the real requirement here. I think that would include things like Dragonfly/Honeybee, but please deploy the Ladybug Tools first.

**Comment by ksobon** [comment]
@michaldengusiak I was able to cleanup the Curtain Wall parsing issues. Basically there were a few things that I didn't anticipate people doing: 
- Nesting Curtain Walls into Curtain Walls as Panels. So basically your Curtain Wall, has bunch of Panels, but instead of those being set as Panels, they were set as another Curtain Wall, which then had Panels. I converted our routines that were extracting panels into recursive functions to keep digging until it gets all of the panels from all nested walls. 
- Another issue were the big Curtain Walls, with just a massive single Panel. In reality that's not physically possible to build a Curtain Wall that would have just one gigantic piece of glass like that, but for the sake of an argument, I added some extra routines into our parser to get these as well. 

The result is a pretty good representation of Glazing from your Revit model, to our Honeybee model: 

![image](https://user-images.githubusercontent.com/529781/85872872-fa906a80-b795-11ea-916e-0f905859e383.png)

Now, that Roof...well, I am not sure I want to go there. Any thoughts? This is obviously somehow parsed by your SAM modeler. Anything that you can share to get me started in the right direction? :-) 

Cheers!

**Comment by chriswmackey** [comment]
@michaldengusiak  , the installation process has undergone a few iterations over the last week or two but I think it is becoming more stable now. We are phasing out the dragonfly installer in favor of this Ladybug Tools installer:
https://github.com/ladybug-tools/lbt-grasshopper/blob/master/installer.gh

I would recommend deleting the ladybug_tools folder in you user folder and follow the instructions again here:
https://github.com/ladybug-tools/lbt-grasshopper/wiki/1.1-Windows-Installation-Steps

**Comment by ksobon** [comment]
@michaldengusiak please give this another go, and we can go and then open a new issue if you encounter anything else.
